src/omuse/community/pop/interface.f90

```diff
@@ -1811,6 +1864,23 @@ function change_directory(pathname) result (ret)
   ret=0
 end function
 
+function set_shf_data_type(type_) result (ret)
+  integer :: ret
+  character (char_len), intent(in) :: type_
+  ret=0
+  if (type_ == 'file')then
+    shf_data_type = type_
+  elseif(type_ == 'amuse') then
+    shf_data_type = type_
+    shf_formulation = 'restoring' 
+    shf_interp_freq = 'every-timestep'
+    shf_interp_type = 'linear'
+  elseif(type_ == 'amuse') then
```

Comment:
this repeats condition...

Reply:
Should be combined into elseif(type_=='amuse' .or. type_=='analytic')